source/2019-09-20-ember-3-13-released.md

```diff
@@ -0,0 +1,190 @@
+
+If you're not an addon author or maintain a component manager, then you are unlikely to need use this new feature.
+
+#### Deprecations (1)
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+For more details on changes in Ember.js 3.13, please review the [Ember.js 3.13.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.13.0).
+
+**Deprecate support for mouseEnter/Leave/Move Ember events (1 of 1)**
+
+Ember.js 3.13 deprecates support for `mouseEnter`, `mouseLeave` and `mouseMove` Ember events. [The deprecation guide](https://deprecations.emberjs.com/v3.x#toc_action-mouseenter-leave-move) contains guidance on how to migrate away from these events.
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.13
+
+#### New Features (0)
+
+No new features introduced in Ember Data 3.13.
+
+#### Deprecations (0)
+
+No new deprecations introduced in Ember Data 3.13.
+
+For more details on changes in Ember Data 3.13, please review the
+[Ember Data 3.13.0 release page](https://github.com/emberjs/data/releases/tag/v3.13.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI easily using the ember-cli-update project:
+
+```bash
+npm install -g ember-cli-update
+ember-cli-update
+```
+
+This utility will help you to update your app or add-on to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the github README](https://github.com/ember-cli/ember-cli-update).
+
+While it is recommended to keep Ember CLI versions in sync with Ember and Ember Data, this is not required. After updating ember-cli, you can keep your current version(s) of Ember or Ember Data by editing `package.json` to revert the changes to the lines containing `ember-source` and `ember-data`.
+
+### Changes in Ember CLI 3.13
+
+No new features in Ember CLI 3.13.
+
+---
+
+For more details on the changes in Ember CLI 3.13 and detailed upgrade
+instructions, please review the [Ember CLI  3.13.0 release page](https://github.com/ember-cli/ember-cli/releases/tag/v3.13.0).
+
+## Ember 3.12 is an LTS Release
+
+If you are upgrading from the previous LTS version (3.8) to 3.12 then you are getting access to these features:
+
+* `router` service 
+* Support for native classes
```

Comment:
ES classes have been supported since August 2017, when [RFC 240](https://github.com/emberjs/rfcs/pull/240) was accepted.

Suggest saying [Native Class Constructor Update RFC](https://github.com/emberjs/rfcs/blob/master/text/0337-native-class-constructor-update.md) implemented

Reply:
Native classes were not officially supported until v3.6 (RFC implementation was incomplete + redesign occurred, no blog post announcement, no API docs). 3.10 adds decorators, which made native classes viable/recommendable for most users, so maybe it should say "Support for native classes + decorators"